README.md

```diff
@@ -18,7 +18,9 @@ Dependencies:
 * [eosio.cdt v1.7.x](https://github.com/EOSIO/eosio.cdt/releases/tag/v1.7.0)
 * [eosio v2.0.x](https://github.com/EOSIO/eos/releases/tag/v2.0.1) (optional dependency only needed to build unit tests)
 
-To build the contracts follow the instructions in [`Build and deploy` section](./docs/02_build-and-deploy.md).
+## Build
+
+To build the contracts follow the instructions in [Build and deploy](https://developers.eos.io/manuals/eosio.contracts/latest/build-and-deploy) section.
```

Comment:
FYI: this should be
https://developers.eos.io/manuals/eosio.contracts/`v1.9`/build-and-deploy, it should hot contain `latest` because this fix specific for `v1.9` version, however because of [this current issue](https://github.com/EOSIO/thorka/issues/231) we can not use the correct link.

Reply:
Agreed - it is what it is right now